testing/docker/run-controlplane-docker.sh

```diff
@@ -20,6 +20,9 @@ set -x
 SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 if [ "$1" == "start" ]; then
+    # Increase memory limit for elasticsearch 5.1
+    sudo sysctl -w vm.max_map_count=262144
+
     echo "starting Control plane components (registry, and controller)"
     docker-compose -f $SCRIPTDIR/controlplane.yaml up -d
```

Comment:
Do we even need this file, as it is no longer doing anything special. I suggest we simply add the following two lines to test-docker.sh
```bash
sysctl -w vm.max_map_count=...
docker-compose -f $SCRIPTDIR/controlplane.yaml up -d
```

Similarly, we can get rid of cleanup.sh and move the two lines into that file. Given that the chances of errors in starting the control plane services is low in travis, we can do away with all the tests (loop, check for health, etc.). May be have it commented. WDYT?